docs/hugo/content/tools/asoctl.md

```diff
@@ -28,45 +28,66 @@ Available Commands:
 
 {{< tabpane text=true left=true >}}
 {{% tab header="**OS**:" disabled=true /%}}
-{{% tab header="Linux" %}}
+{{% tab header="Linux (incl WSL)" %}}
 
-AMD64:
+Download for AMD64:
 
-`curl -L https://github.com/Azure/azure-service-operator/releases/latest/download/asoctl-linux-amd64 -o asoctl`
+``` bash
+$ curl -L https://github.com/Azure/azure-service-operator/releases/latest/download/asoctl-linux-amd64.gz -o asoctl.gz
```

Comment:
1. is `gunzip` standard? I feel like I usually see a `tar ...` cmd here.
2. The other thing is, can we do this w/ a pipe rather than 2 cmds? So that it's a one-liner? maybe refer to some example repos to see if they do it as a one-liner or not.